config/v1/types_apiserver.go

```diff
@@ -73,6 +73,10 @@ const (
 	// "AllRequestBodies" is similar to WriteRequestBodies, but also logs request
 	// and response HTTP payloads for read requests (get, list).
 	AllRequestBodiesAuditProfileType AuditProfileType = "AllRequestBodies"
+
+	// "UserRequests" is a policy that filters logs on resource requests from
```

Comment:
`UserRequestBodies` ?

Reply:
Hey @tkashem  and @awgreene :waiting to hear about scope of the work that maybe changing. Enum key itself may need changing.